tests/data/generated/transformations_v2.yaml

```diff
@@ -13,582 +13,17 @@ transformations:
     parameters:
       relative_datapoints:
       - offset_minute: 0.0
-        offset_value: 100000.0
-- StaticValues:
```

Comment:
Earlier bug?

Reply:
Rather that only the unique ones are written to this file now, so duplicates are removed. Also, this file is not used by resync in the end (only the new time series mappings and price scenarios files). It is just something I added to be able to be able to do some verification and sanity checks